### PR TITLE
Done

### PR DESCRIPTION
**Pull request by MaksimusCRS**
[DEMO LINK](https://MaksimusCRS.github.io/react_movies-list-fetch-movies/)

### CODE REVIEW

**Review by Oleksandr-Leshchenko — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
-  state: State = {
-    movies: [],
-  };
+export const App: React.FC = () => {
+  const [movies, setMovies] = useState<Movie[]>([]);
 
-  render() {
-    const { movies } = this.state;
+  const addMovie = (movie: Movie): void => {
+    const includeMovie = movies.some(({ imdbID }) => movie.imdbID === imdbID);
```
Comment (Elena-brv) [inline src/App.tsx:11]:
```suggestion
    const isMovieExist = movies.some(({ imdbID }) => movie.imdbID === imdbID);
```

File: src/components/FindMovie/FindMovie.tsx [hunk 2]
```diff
@@ -1,54 +1,105 @@
...
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [error, setError] = useState(false);
+  const [movie, setMovie] = useState(null);
+  const [title, setTitle] = useState('');
+
+  const searchMovie = () => {
+    getMovie(title)
+      .then(newMovie => {
```
Comment (Elena-brv) [inline src/components/FindMovie/FindMovie.tsx:20]: use async/await

File: src/components/FindMovie/FindMovie.tsx [hunk 3]
```diff
@@ -1,54 +1,105 @@
...
+
+            <div className="control">
+              <input
+                value={title}
+                onChange={handleInput}
+                type="text"
+                id="movie-title"
+                placeholder="Enter a title to search"
+                className={classNames('input',
+                  { 'input is-danger': error })}
```
Comment (Elena-brv) [inline src/components/FindMovie/FindMovie.tsx:64]:
```suggestion
                className={classNames(
                  'input',
                  { 'input is-danger': error }
                )}
```

File: src/components/FindMovie/FindMovie.tsx [hunk 4]
```diff
@@ -1,54 +1,105 @@
...
-
-    <div className="container">
-      <h2 className="title">Preview</h2>
-      {/* <MovieCard  /> */}
-    </div>
-  </>
-);
+    </>
+
+  );
```
Comment (Elena-brv) [inline src/components/FindMovie/FindMovie.tsx:104]:
```suggestion
    </>
  );
```

**Review by Elena-brv — APPROVED** [review]

**Pull request closed without merging**